### PR TITLE
Add tags to read role

### PR DESCRIPTION
**Pull request by LumaC0**
# Description

Changes the return value of getAccountRole from response.roleArn of type string to a role object that better reflects the response from ALKS core. This change is centered around recent AWS resource tagging functionality in ALKS.

This PR also creates a change-log and adds this very same PR template

## Type of change

-  Breaking change (fix or feature that would cause existing functionality to not work as expected)
Code expecting a string  return type for getAccountRole will have to be updated to accept a Role object

# How has this been tested?

1. npm test

# Checklist / To Do:

- Write documentation reflecting the change
I was unsure how to proceed wit typedoc so I left it unmodified.

### CODE REVIEW

File: src/alks.ts [hunk 1]
```diff
@@ -152,12 +152,15 @@ namespace ALKS {
    * @property denyArns - The ARNs for the deny policies associated with this role
    * @property instanceProfileArn - The Instance Profile ARN associated with this role
    * @property addedRoleToInstanceProfile - Whether this role was added to an Instance Profile
+   * @property RoleTags - The tags associated with the IAM account role
```
Comment (elliottzack429) [inline src/alks.ts:155]: Shouldnt this just be called 'tags' since thats what you named it in the interface?

File: src/alks.ts [hunk 2]
```diff
@@ -204,6 +207,16 @@ namespace ALKS {
     machineIdentityArn: string;
   }
 
+  /**
+   * Alks role tags
+   * @property tagKey - The AWS role tag key
```
Comment (elliottzack429) [inline src/alks.ts:212]: Same here, seems like convention is for the @property to have the same name as the actual property in the object

File: src/alks.ts (hunk 1 above)
Comment (LumaC0) [reply src/alks.ts:155]: I forgot to change it when I changed the object name. I didn't run typedoc so this is localized within the source code

File: test/test.js [hunk 3]
```diff
@@ -464,10 +468,44 @@ describe('alks.js', function () {
...
+        .mock('https://your.alks-host.com/getAccountRole', {
+          body: {
+            roleArn:
+              'arn:aws:iam::12391238912383:role/acct-managed/awsRoleName',
+            roleName: 'awsRoleName',
+            basicAuthUsed: false,
+            roleExists: true,
+            instanceProfileARN: 'anInstanceProfileArn',
+            isMachineIdentity: false,
+            tags: [{ key1: 'test:key1:value1', key2: 'test:key2:value1' }],
```
Comment (elliottzack429) [inline test/test.js:489]: Doesnt Alks core return:
tags [
    {
      key: "key1",
      value: "test:key1:value1"
    },
   {
     key: "key2",
     value: "test:key2:value1"
   }
]

**Review by elliottzack429 — APPROVED** [review]

**Pull request merged**